integrations/oci/metrics/metrics/src/main/java/module-info.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2022 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Integration with OCI Metrics.
+ */
+module io.helidon.integrations.oci.metrics {
+    requires java.logging;
+
+    requires transitive io.helidon.config;
+    requires transitive io.helidon.common;
+    requires io.helidon.common.http;
+    requires io.helidon.webserver;
+    requires io.helidon.config.metadata;
```

Comment:
I think I steered you wrong earlier. This should be `requires static` IIRC.

Reply:
Updated in the most recent commit.